api/endpoints/logged_activities.py

```diff
@@ -0,0 +1,34 @@
+from flask import jsonify, request
+from flask_restplus import Resource
+
+from api.models import LoggedActivity, User
+from api.utils.auth import token_required
+from api.utils.marshmallow_schemas import get_logged_activities_schema
+
+
+class LoggedActivitiesAPI(Resource):
+    """Logged Activities Resources."""
+    decorators = [token_required]
+
+    def get(self):
+        """Get a user's logged activities by user URL query"""
+        user_id = request.args.get('user', type=str)
+        if not user_id:
+            return {"message": "No URL queries could be parsed"}, 400
+
```

Comment:
@LarryWachira Shouldn't these checks be implemented in a validator function in marshmallow? I assume that was the process.